packages/code-du-travail-ui/package.json

```diff
@@ -4,49 +4,67 @@
   "private": true,
   "description": "Composants ReactJS du Code du travail numérique",
   "main": "lib/index.js",
-  "module": "es/index.js",
+  "module": "lib/index.esm.js",
   "files": [
-    "css",
     "es",
-    "lib",
-    "umd"
+    "lib"
   ],
   "scripts": {
-    "build": "nwb build-react-component --no-demo ",
-    "clean": "nwb clean-module && nwb clean-demo",
-    "lint": "eslint src/* tests/*",
+    "build": "NODE_ENV=production rollup --config",
+    "lint": "eslint src/*",
     "prepush": "yarn lint && yarn test",
     "prepublishOnly": "npm run build && npm run gh-pages",
     "gh-pages": "npm run docz:build && gh-pages -d ./doc-build",
     "start": "docz dev",
-    "test": "nwb test-react",
-    "test:coverage": "nwb test-react --coverage",
-    "test:watch": "nwb test-react --server",
-    "docz:build": "docz build"
+    "docz:build": "docz build",
+    "test": "jest"
   },
   "peerDependencies": {
     "prop-types": "^15.6.2",
-    "react": "16.x",
-    "styled-components": "^4.1.2"
+    "react": "^16.2.0"
```

Comment:
Should match the current version in dev dependency or match all major version.
Peer dependencies are hard to handle. The less constrained the most effective there are

Reply:
thanks, I'm not familiar with peerDep